samples/CSharp/Analyzers/Analyzers.Test/Analyzers.CSharp.Tests.csproj

```diff
@@ -1,11 +1,16 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <TargetFramework>net46</TargetFramework>
```

Comment:
This should be unnecessary now that we target .NET Standard 2.0

Reply:
This change was reverted.